Sakura/core/payments.py

```
@@ -0,0 +1,116 @@
+        await context.bot.send_sticker(chat_id=update.message.chat.id, sticker=sticker_id)
+        await asyncio.sleep(4)
+        keyboard = [[InlineKeyboardButton("Buy flowers again 🌸", callback_data="get_flowers_again")]]
+        reply_markup = InlineKeyboardMarkup(keyboard)
+        success_msg = random.choice(THANK_YOU_MESSAGES)
+        if update.message.chat.type == "private":
+            try:
+                url = f"https://api.telegram.org/bot{BOT_TOKEN}/sendMessage"
+                payload = {
+                    'chat_id': update.message.chat.id,
+                    'text': success_msg,
+                    'message_effect_id': random.choice(EFFECTS),
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)